.github/workflows/CI.yaml

```diff
@@ -19,12 +19,10 @@ env:
       - '.editorconfig'
       - 'docs/**'
       - 'example/*.png'
-
-  pull_request: null
+  pull_request:
 
 jobs:
   build:
-    if: "!contains(github.event.head_commit.message, 'skip ci')"
```

Comment:
This is now built-in: https://github.blog/changelog/2021-02-08-github-actions-skip-pull-request-and-push-workflows-with-skip-ci/

Reply:
Nice!